src/apps/chifra/pkg/rpcClient/client.go

```diff
@@ -99,6 +100,29 @@ func GetIDs(provider string) (uint64, uint64, error) {
 	return ch.Uint64(), n.Uint64(), nil
 }
 
+// TODO: C++ code used to cache version info
```

Comment:
We tried to cache everything we ever asked of the node. The node is SLOW. This brings up an interesting issue. There's a global option called `--cache` which, if one, should cache the returned value from the node. If that returned value is present, we should read it. Super simple cache since we can safely assume the data is immutable (if it's old enough -- six minutes?)